querybook/webapp/components/CodeMirrorTooltip/SuggestionTooltip.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import { Button } from 'ui/Button/Button';
+
+interface IProps {
+    suggestionText: string;
+    onAcceptSuggestion: () => void;
+    onCancel: () => void;
+}
+
+const SUGGESTION_MAX_LENGTH = 20;
```

Comment:
is 20 long enough to handle the 3 use cases?

Reply:
It's long enough to at least show the keyword that's being changed for each of the cases. (Otherwise, I thought the string could get pretty long for the `REGEXP_LIKE` suggestion)